queries/vue/injections.scm

```diff
@@ -8,7 +8,7 @@
         (attribute_value) @injection.language)))
   (raw_text) @injection.content)
   (#eq? @_lang "lang")
-  (#any-of? @injection.language "css" "scss"))
+  (#any-of? @injection.language "css" "scss" "sass"))
```

Comment:
No. Only one of the two places. Only the one below should be enough

Reply:
@lucario387 done